scss/_footer.scss

```diff
@@ -0,0 +1,166 @@
+  letter-spacing: $footer-letter-spacing;
+
+  @include media-breakpoint-up(md) {
+    flex-direction: row;
+  }
+
+  .nav-link {
+    display: flex;
+    align-items: center;
+    padding: 0;
+    outline-offset: map-get($spacers, 1);
+
+    &.active::before {
+      content: unset;
+    }
+  }
+}
+
+.footer-heading {
+  font-size: $footer-font-size-md;
+  font-weight: $font-weight-bold;
+  line-height: $footer-line-height-md;
+  letter-spacing: $footer-letter-spacing;
+}
+
+.footer-title-content {
+  padding-top: $footer-mailing-padding-top;
+  padding-bottom: $footer-mailing-padding-bottom;
+
+  @include media-breakpoint-up(md) {
+    padding-bottom: $footer-mailing-padding-bottom-md;
+  }
+
+  .footer-heading {
+    margin-bottom: $footer-title-margin-bottom;
+  }
+}
+
+.footer-social {
+  display: flex;
+  flex-direction: column;
+  padding-top: $footer-social-padding-top;
+  padding-bottom: $footer-social-padding-bottom;
+
+  @include media-breakpoint-up(md) {
+    flex-direction: row;
+    padding-top: $footer-social-padding-top-md;
+  }
+
+  > .footer-heading {
+    margin: $footer-social-title-margin-md;
+    white-space: nowrap;
+
+    @include media-breakpoint-down(md) {
+      margin: 0 0 $footer-title-margin-bottom;
+      font-size: $footer-font-size-sm;
+      line-height: $footer-line-height-sm;
+    }
+  }
+}
+
+.footer-nav {
+  @include media-breakpoint-down(md) {
+    max-width: 100vw;
+    padding: 0;
+
+    .row { // Here to replace overflow-x on parent that allow focus on parent.
+      margin-right: 0;
+      margin-left: 0;
+
+      > * {
+        padding-right: 0;
+        padding-left: 0;
+      }
+    }
+  }
+
+  .footer-column:first-of-type .accordion-header:first-of-type,
+  .accordion .accordion-item {
+    border: 0;
+  }
+
+  .accordion-header {
+    @include media-breakpoint-up(md) {
+      padding: $footer-nav-padding-top 0 $spacer;
+      border: 0;
+    }
+
+    .accordion-button {
+      font-size: $footer-font-size-sm;
+      line-height: $footer-accordion-line-height;
+      letter-spacing: $footer-letter-spacing;
+
+      &::after {
+        margin-right: 0;
+      }
+
+      @include media-breakpoint-up(md) {
+        all: unset;
+        pointer-events: none;
+
+        &::after {
+          content: unset;
+        }
+      }
+    }
+  }
+
+  .navbar-nav {
+    gap: $footer-gap;
+    padding: $footer-nav-list-padding;
+
+    @include media-breakpoint-up(md) {
+      flex-direction: column;
+      padding-bottom: $footer-nav-list-padding-bottom;
+      font-size: $footer-font-size-md;
+      line-height: $footer-line-height-md;
+    }
+
+    @include media-breakpoint-up(lg) {
+      padding-bottom: $footer-nav-list-padding-bottom-md;
+    }
+  }
+
+  .accordion-collapse {
+    @include media-breakpoint-up(md) {
+      display: flex;
+      padding-left: 0;
+    }
+  }
+}
+
+.footer-service {
+  padding-top: subtract($footer-service-padding-y, .0625rem);
+  padding-bottom: $footer-service-padding-y;
+
```

Comment:
Question: Do we need to force icon size here by adding `width: 1.857rem`?

Reply:
I would stick with what's done in `docs/5.1/components/buttons/#icon-only` and let the users define the the width/height.